### PR TITLE
Revert "ci: update to latest Core Operator version"

### PR DESCRIPTION
**Pull request by jasmingacic**
Reverts chronosphereio/calyptia-cli#889

### CODE REVIEW

**Review by patrick-stephens — CHANGES_REQUESTED** [review]
We should not be changing the registry, plus it has a typo.
Similarly we should keep versions rather than `latest`.

File: cmd/utils/utils.go [hunk 1]
```diff
@@ -9,12 +9,8 @@ import (
 )
 
 const (
-	LatestVersion = "latest"
-	//nolint: gosec // this is not a secret leak, it's just a format declaration.
-	DefaultCoreDockerImage    = "ghcr.io/calyptia/core"
```
Comment (patrick-stephens) [inline cmd/utils/utils.go:14]: Why are we dropping this?

File: cmd/utils/utils.go [hunk 2]
```diff
@@ -9,12 +9,8 @@ import (
...
-	DefaultCoreDockerImageTag = LatestVersion
-
-	DefaultCoreOperatorDockerImage = "ghcr.io/calyptia/core-operator"
+	LatestVersion                  = "latest"
+	DefaultCoreOperatorDockerImage = "ghcr.io/chronoshpereio/core-operator"
```
Comment (patrick-stephens) [inline cmd/utils/utils.go:13]: No, let's keep the current registry until we have properly covered migration in docs/tests/etc.
Plus this has a typo

File: cmd/operator/install.go [hunk 3]
```diff
@@ -196,6 +196,7 @@ func prepareInstallManifest(coreDockerImage, coreInstanceVersion, namespace stri
 
 	withImage, err := addImage(coreDockerImage, coreInstanceVersion, withNamespace)
 	if err != nil {
+		fmt.Println("!!!!")
```
Comment (patrick-stephens) [inline cmd/operator/install.go:199]: Might be a better debug output? :)

Comment (jasmingacic) [reply cmd/operator/install.go:199]: i love to put it like that :D

File: cmd/utils/utils.go (hunk 2 above)
Comment (jasmingacic) [reply cmd/utils/utils.go]: done

File: cmd/utils/utils.go (hunk 1 above)
Comment (jasmingacic) [reply cmd/utils/utils.go:14]: it's not used in the code

**Review by patrick-stephens — APPROVED** [review]

**Pull request merged**